### PR TITLE
ISPN-7544 Server cards do not indicate correct status in some cases

### PR DESCRIPTION
**Pull request by vblagoje**
Master only. See https://issues.jboss.org/browse/ISPN-7544 for more details. Test using steps described in https://issues.jboss.org/browse/ISPN-7462

### CODE REVIEW

File: src/services/container/ContainerService.ts [hunk 1]
```diff
@@ -146,7 +146,7 @@ export class ContainerService {
     // Otherwise, ensure that the views of all nodes within the server group are the same. Is this second step necessary?
     this.$q.all(statusPromises)
       .then((statuses: string[]) => {
-        return statuses.indexOf("STOPPED") === -1;
+        return statuses.indexOf(SERVER_STATE_STOPPED) === -1;
```
Comment (ryanemerson) [inline src/services/container/ContainerService.ts]: SERVER_STATE_STOPPED import is missing in this class so the app cannot load

Comment (vblagoje) [inline src/services/container/ContainerService.ts]: +1 crazy...thanks

**Comment by vblagoje** [comment]
It should work now @ryanemerson

**Comment by ryanemerson** [comment]
@vblagoje Integrated https://github.com/infinispan/infinispan-management-console/commit/adf10bc534928c1272453e3d7ac801447ff0d86e thanks!

**Pull request closed without merging**